modules/nodes-descheduler-configuring-profiles.adoc

```diff
@@ -34,14 +34,16 @@ spec:
   logLevel: Normal
   managementState: Managed
   operatorLogLevel: Normal
-  profiles:
-  - AffinityAndTaints       <1>
-  - TopologyAndDuplicates   <2>
-  - LifecycleAndUtilization <3>
+  profileCustomizations:
+    podLifetime: 30m        <1>
+  profiles:                 <2>
+  - AffinityAndTaints
+  - TopologyAndDuplicates   <3>
+  - LifecycleAndUtilization
```

Comment:
@damemi do you think we ought to add any of the new profile examples (local storage and PVC) here to this example? I listed them all in the callout below, but wasn't sure what made sense to show in the example here.